lib/web_ui/lib/src/engine/canvaskit/image_web_codecs.dart

```diff
@@ -7,36 +7,20 @@
 /// See also:
 ///
 ///  * `image_wasm_codecs.dart`, which uses codecs supplied by the CanvasKit WASM bundle.
-@JS()
 library image_web_codecs;
```

Comment:
Now that you removed `@JS()`, I don't think you need the `library` directive, do you?

Reply:
Removed.